Resources/Prototypes/SimpleStation14/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -2,7 +2,7 @@
   name: pv3 compact
   parent: BaseWeaponBattery
   id: WeaponLaserPv3Compact
-  description: Standard issue Nyanotrasen laser SMG. Pew! Pew pew! Now we're talkin' scifi!
+  description: Standard issue Nanotrasen laser SMG. Pew! Pew pew! Now we're talkin' scifi!
```

Comment:
```suggestion
  description: Standard issue NanoTrasen laser SMG. Pew! Pew pew! Now we're talkin' scifi!
```

Reply:
this is very wishy washy on whether or not it is used. if this is being enforced i'd suggest it done in one big sweep as there would be a lot to change

Reply:
NT - **N**ano**T**rasen

Reply:
NT is an abbreviation, they tend to use uppercase even if they use middle of word letters yes